Screencast

erikhopf opened the pull request:
Adding screencast to Getting Started.

@kellyamanda - 2 quick questions:

* How do you feel about the current sizing of the animated gif? Would you prefer if I zoomed in a bit to highlight the content of the report?
* I used the output of the tutorial for this screengrab. Let me know if you'd like me to use another animation that's more pronounced -- or pops. It shouldn't take very long.

tvst commented on docs/getting_started.md:
```diff
@@ -288,6 +288,18 @@ for i in range(100):
 
+## Record a screencast
+
+After you've built a Streamlit report, it's easy to share. With screencast, you can record your interactions as you move through a report and add a voice over to explain what you're doing. When you're finished, you can stop the recording, save it and share it.
```
"report" -> "app"

tvst replied:
I think a better intro would be more like: (please edit)

> After you've built a Streamlit app, you may want to discuss some of it with co-workers over email or Slack, or share it with the world on Twitter. A great way to do that is with Streamlit's built-in screencast recorder. With screencast, you can (...)

tvst commented on docs/getting_started.md:
```diff
@@ -288,6 +288,18 @@ for i in range(100):
+
+After you've built a Streamlit report, it's easy to share. With screencast, you can record your interactions as you move through a report and add a voice over to explain what you're doing. When you're finished, you can stop the recording, save it and share it.
+
+To start a screencast, locate the menu in the upper right corner of your report (**☰**), select **Record a screencast**, and follow the prompts. Before the recording starts, you'll see a countdown -- this means it's showtime.
```
Nit: "--" should be "—"

tvst replied:
> this means it's showtime.

Love this!

tvst replied:
report -> app

tvst commented on docs/getting_started.md:
```diff
@@ -288,6 +288,18 @@ for i in range(100):
+
+To start a screencast, locate the menu in the upper right corner of your report (**☰**), select **Record a screencast**, and follow the prompts. Before the recording starts, you'll see a countdown -- this means it's showtime.
+
+To stop your screencast, go back to the menu (**☰**) and select **Stop recording** (or hit the **ESC** key). Follow the prompts to preview your recording and save it to disk. That's it, you're ready to share your Streamlit report.
```
report -> app

erikhopf commented:
@tvst - Hey Thiago, sorry for the large delay on this. I've addressed your concerns and compressed the image. It's still a bit large, but I should be able to address this in the next patch. 

CC - @kellyamanda

tvst commented:
Looks good. I made a couple of small changes  https://github.com/streamlit/streamlit/pull/1142/commits/0670b005a0265058487c5337f3d373432b45ca69 and will merge as soon as the tests pass
